documentation/Generate-code-from-TypeSpec.md

```diff
@@ -0,0 +1,113 @@
+# Before you Start
+
+[TypeScript Azure SDK Design Guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) is the overall design guideline of the client SDK.
```

Comment:
Do we have short url?

Reply:
Agree, suggest to use short link especially for the links to the github repo.